geometry_msgs/Pose thumb_tip
geometry_msgs/Pose index_tip
geometry_msgs/Pose middle_tip
geometry_msgs/Pose ring_tip
geometry_msgs/Pose pinky_tip
